androidApp/src/main/java/dev/johnoreilly/confetti/auto/speakers/SpeakersScreen.kt

```diff
@@ -15,86 +17,117 @@ import dev.johnoreilly.confetti.SpeakersUiState
 import dev.johnoreilly.confetti.SpeakersViewModel
 import dev.johnoreilly.confetti.auto.speakers.details.SpeakerDetailsScreen
 import dev.johnoreilly.confetti.auto.ui.ErrorScreen
-import dev.johnoreilly.confetti.auto.utils.AutoImage
 import dev.johnoreilly.confetti.auto.utils.fetchImage
-import dev.johnoreilly.confetti.auto.utils.getDefaultBitmap
 import dev.johnoreilly.confetti.fragment.SpeakerDetails
-import kotlinx.coroutines.launch
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.flatMapLatest
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.flow.stateIn
 
 class SpeakersScreen(
     carContext: CarContext,
     conference: String,
-    private val speakersViewModel: SpeakersViewModel = SpeakersViewModel()
+    speakersViewModel: SpeakersViewModel = SpeakersViewModel()
 ) : Screen(carContext) {
 
-    private var images: MutableList<AutoImage> = mutableListOf()
-    private var uiState: SpeakersUiState = SpeakersUiState.Loading
+    val fallbackImage = CarIcon.Builder(
```

Comment:
Small thing here but:
- `fallbackImage`, `speakersFlow` and `imagesFlow` could be set as private